.github/workflows/build_hub.yml

```diff
@@ -141,5 +156,37 @@ jobs:
         run: |
           false
 
+  build_docker:
+    name: Build docker image
+    needs:
+      - build_router
+      - linting
+      - docs
+    if: ${{ needs.pre_job.outputs.has_router_change == 'true' }}
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: prime-router
+
+    steps:
+      - name: "Check out changes"
+        uses: actions/checkout@v2
+
+      - name: Set Environment Variables
+        run: |
+          if [ "$GITHUB_BASE_REF" == "production" ]
+          then
+              echo "Building for the production environment."
+              echo >> $GITHUB_ENV ACR_REPO=pdhprodcontainerregistry.azurecr.io
+              echo >> $GITHUB_ENV PREFIX=pdhprod
+          else
+              echo "Building for the test environment."
+              echo >> $GITHUB_ENV ACR_REPO=pdhstagingcontainerregistry.azurecr.io
+              echo >> $GITHUB_ENV PREFIX=pdhstaging
+          fi
+
```

Comment:
We can actually drop these lines if you if you update line 192 to be something like `CDCgov/reportstream`. The actual image tag name only matters in the release job, so it's kind of pointless for this extra logic in the CI job.

Reply:
ok makes total sense, that's why I was wondering if it was just checking the build.